bin/collator/src/local/chain_spec.rs

```diff
@@ -6,6 +6,7 @@ use local_runtime::{
     VestingConfig,
 };
 use sc_service::ChainType;
+use sp_core::crypto::Ss58Codec;
```

Comment:
Can be done when polishing but dependencies should be grouped together.

E.g. use sp_core::{crpyto::Ss58Codec, sr25519, Pair, Public};

Reply:
done